.env.example

```diff
@@ -4,6 +4,9 @@
 PORT=3000
 SESSION_SECRET="9f86d081884c7d659a2feaa0c55ad015a3bf4f1b2b0b822cd15d6c15b0f00a08"
 REDIS_URL="redis://redis:6379"
+SMPT_USERNAME="noreplay.teamupapi@gmail.com"
```

Comment:
typo: ~~SMPT~~ SMTP
typo: ~~noreplay~~ noreply

is this sensitive data??? 0_0